src/components/SearchContainer/SearchContainer.tsx

```diff
@@ -0,0 +1,32 @@
+import { useState } from "react";
+
+function SearchContainer({ handleIncrement, handleSubmit }: any) {
+  const [searchEntry, setSearchEntry] = useState("");
+  const updateSearchInput = (event: any) => {
+    setSearchEntry(event.target.value);
+  };
+  return (
+    <form
+      className="searchContainer"
+      onSubmit={(event) => {
```

Comment:
Usually it is better to have all the handlers as a separate functions, as `updateSearchInput`, for example, because eslint throws an error about this by default.